tests/_test_msui/test_msui.py

```diff
@@ -328,3 +328,225 @@ def test_flight_track_io(self, mockload, mocksave, mockq, mocki, mockw):
         assert self.window.listFlightTracks.count() == 2
         assert os.path.exists(self.save_ftml)
         os.remove(self.save_ftml)
+
+class Test_MSUIMainWindow:
+    def setUp(self):
```

Comment:
This is a unittest-style test class, but we do pytest. Please turn this into an autouse fixture, then you can also apply Reimar's recommendation to use the tmp_path fixture (which wouldn't work with the unittest-style right now).